buildtest/main.py

```diff
@@ -165,6 +166,8 @@ def main():
             summarize_buildspec_cache(configuration)
         elif args.buildspecs_subcommand == "show":
             show_buildspecs(test_names=args.name, configuration=configuration)
+        elif args.buildspecs_subcommand == "show-fail":
+            show_failed_buildspecs(configuration=configuration)
```

Comment:
please pass argument `report_file` this is defined in this file see https://github.com/buildtesters/buildtest/blob/devel/buildtest/main.py#L74